legacy/src/api/merchants/merchant-configs.md

```diff
@@ -59,7 +59,7 @@ A Merchant Config defines the available payment options for paying a [Payment Re
 | paypalMerchantAccountId | String | Id of the PayPal merchants account. Required for `paypal.usd` and `venmo.usd` types.                                                                                                 |
```

Comment:
Just noticed these asset types should have been changed when we removed the `usd` currency from them. Do you mind changing this is a follow-up PR?